code-reviews/recipes/CSharp.md

```diff
@@ -77,6 +77,12 @@ If you are using FxCop analyzers and StyleCop analyzer, it's very simple to enab
         projects: '**/*.csproj'
 ```
 
+## Enable Roslyn Support in Visual Studio Code
+
+The above steps also work in vscode provided you enable Roslyn support for Omnisharp. The setting is "omnisharp.enableRoslynAnalyzers" and must be set to "true". After enabling this setting you must "Restart Omnisharp" (this can be done from the Command Palatte in vscode or by restarting vscode).
```

Comment:
```suggestion
The above steps also work in VS Code provided you enable Roslyn support for Omnisharp. The setting is `omnisharp.enableRoslynAnalyzers` and must be set to `true`. After enabling this setting you must "Restart Omnisharp" (this can be done from the Command Palette in VS Code or by restarting VS Code).
```